geometry/distance.c

```diff
@@ -0,0 +1,37 @@
+#include <math.h>
+#include <stdio.h>
+/**
+ * Author Arnab Nath
+ * The code is implemented in VS Code
```

Comment:
You should explain what is the algorithm about, how it works, a Wikipedia link, space complexity, etc.. etc..